vllm/model_executor/models/ultravox.py

```diff
@@ -466,9 +471,9 @@ def forward(self, input_ids: torch.Tensor, positions: torch.Tensor,
                 inputs_embeds = self.language_model.model.get_input_embeddings(
                     input_ids)
 
-                inputs_embeds = merge_multimodal_embeddings(
-                    input_ids, inputs_embeds, audio_embeddings,
-                    _AUDIO_PLACEHOLDER_TOKEN)
+                merge_multimodal_embeddings_from_map(
```

Comment:
It looks like "merge_multimodal_embeddings_from_map" is used only for ultravox model, where this kind of merging happens for the other models?

Reply:
See the other comment -- in time I'd expect usage of `merge_multimodal_embeddings` to be replaced with `merge_multimodal_embeddings_from_map`.

Reply:
How can i use it in other multimodal models such as llava? Do ```multi_modal_placeholder_index_maps``` have other keys besides "audio" use for image?